modules/apps/account/account-test/src/testFunctional/tests/Accounts.testcase

```diff
@@ -3433,6 +3433,49 @@ definition {
 		LexiconEntry.viewNoEntry(rowEntry = "Account without Role");
 	}
 
+	@description = "Use case for LPS-139430. Asserts a user can be removed from an account through the Users Admin portlet"
+	@priority = "3"
+	test RemoveUserFromAccountViaUsersAdmin {
+		task ("Given two accounts are created") {
+			for (var accountName : list "Keep Account,Remove Account") {
+				JSONAccountEntry.addAccountEntry(
+					accountEntryName = "${accountName}",
+					accountEntryType = "Business");
+			}
+
+			task ("And given an account user is created and added to both of those accounts") {
+				JSONAccountEntryUser.addAccountEntryUser(
+					accountEntryName = "Keep Account",
+					userEmailAddress = "userea@liferay.com",
+					userFirstName = "userfn",
+					userLastName = "userln",
+					userScreenName = "usersn");
+
+				User.assignAccountCP(
+					accountName = "Remove Account",
+					userScreenName = "usersn");
+			}
+		}
+
+		task ("When removing the second account through the Users Admin") {
+			User.removeAccountCP(
+				accountName = "Remove Account",
+				userScreenName = "usersn");
+
+			Refresh();
+		}
+
+		task ("Then the user should only have the first account listed in Users Admin") {
+			AssertElementPresent(
```

Comment:
This probably can be replace by the existing macro in LexibleTable, would you please double check? Thanks